scipy/stats/morestats.py

```diff
@@ -17,6 +17,7 @@
 from . import distributions
 from ._distn_infrastructure import rv_generic
 from ._hypotests import _get_wilcoxon_distr
+import scipy.stats.stats
```

Comment:
A little nitpick:

```suggestion
from .stats import _normtest_finish
```

Do we want to go so many namespaces deep? `scipy.stats.stats` seems a little redundant. There are other tests like `ansari` which will also use this function. So, maybe do `from .stats import _normtest_finish`?

Reply:
No, I just have never learned how relative imports work and i didn't try anything that I thought might cause circular import problems. This is definitely preferable if it works.